docs/_docs/security/sandbox.adoc

```diff
@@ -0,0 +1,80 @@
+= The Ignite Sandbox
+
+== Overview
+Ignite allows using a user-defined code that can be a compute job, event filter, message listener, etc.
+This user-defined code can utilize Java features to get access to host resources.
+For example, it can create/update/delete files; create sockets; read/update/delete system properties; use reflection API and so on.
+To restrict user-defined code options you may use the Ignite Sandbox based on the Java Sandbox model.
+
+You can read about the Java Sandbox model link:https://docs.oracle.com/en/java/javase/11/security/java-se-platform-security-architecture.html#GUID-C203D80F-C730-45C3-AB95-D4E61FD6D89C[here,window=_blank].
+
+== Ignite Sandbox activation
```

Comment:
Ignite Sandbox activation => Ignite Sandbox Activation

All the letters have to be capitalized (the rule doesn't apply to prepositions and articles)

Reply:
ok